### PR TITLE
feat: added histogram paddle frontend and tests

### LINKED ISSUE #26663: histogram

### PR DESCRIPTION
**Pull request by lukaM19**
# PR Description

## Related Issue

Close #26663

## Checklist

- [ ] Did you add a function?
- [ ] Did you add the tests?
- [ ] Did you run your tests and are your tests passing?
- [ ] Did pre-commit not fail on any check?
- [ ] Did you follow the steps we provided?

### CODE REVIEW

**Comment by lukaM19** [comment]
Hi @jieunboy0516 , the new failures which are introduced are not resalted to the function I implemented so I am a little confused, I would greatly appreciate any tips ! thank you

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py [hunk 1]
```diff
@@ -2322,6 +2381,40 @@ def test_paddle_tensor_greater_than(
     )
 
 
+# histogram
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="paddle.to_tensor",
+    method_name="histogram",
+    values=_histogram_helper(),
```
Comment (jieunboy0516) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py:2389]: `TypeError: _histogram_helper() missing 1 required positional argument: 'draw'`
could you fix that please?

**Comment by jieunboy0516** [comment]
> Hi @jieunboy0516 , the new failures which are introduced are not related to the function I implemented so I am a little confused, I would greatly appreciate any tips ! thank you

Thanks for your contribution @lukaM19 . I dont think your code caused the new failure, should be something else. Please fix your test helper so that i can run the unit test! thx

**Comment by NripeshN** [comment]
Hey @lukaM19,
Could you please fix the merge conflicts

**Pull request closed without merging**